doc/tutorial/index.rst

```diff
@@ -96,8 +96,15 @@ following content.
 
 .. code-block:: text
 
-    docs ├── build ├── make.bat ├── Makefile └── source ├── conf.py ├──
-    index.rst ├── _static └── _templates
+   docs
+   ├── build
+   ├── make.bat
+   ├── Makefile
+   └── source
+      ├── conf.py
+      ├── index.rst
+      ├── _static
+      └── _templates
```

Comment:
How do you feel about adding `/` to make it clearer what's a directory and what's not?

```suggestion
   docs/
   ├── build/
   ├── make.bat
   ├── Makefile
   └── source/
      ├── conf.py
      ├── index.rst
      ├── _static/
      └── _templates/
```

Reply:
This is the unmodified output of `tree`, see https://github.com/sphinx-doc/sphinx/pull/9276#discussion_r642507133

I don't have strong feelings either way, but I'm pinging @stevepiercy because he reviewed this part in the first PR.

Reply:
I prefer the unmodified output of `tree` because it reduces maintenance and the names of directories do not include `/`.

Reply:
Hmm... I was wondering why my tree has that by default, and I found that it's because it's aliased to `tree -F`.

I disagree with some stuff said here but that's water under the bridge now and there's better stuff to spend our collective energy on than trailing slashes! :)